packages/serverpod/lib/src/server/websocket_request_handlers/method_websocket_request_handler.dart

```diff
@@ -160,69 +172,22 @@ class MethodWebsocketRequestHandler {
         connectionId: message.connectionId,
         responseType: OpenMethodStreamResponseType.endpointNotFound,
       );
-    }
-
-    // Parse arguments
-    Map<String, dynamic> args;
-    try {
-      args = EndpointDispatch.parseParameters(
-        message.args,
-        endpointMethodConnector.params,
-        server.serializationManager,
-      );
-    } catch (e) {
-      server.serverpod.logVerbose(
-        'Failed to parse parameters for open stream request: $message',
-      );
-      return OpenMethodStreamResponse.buildMessage(
-        endpoint: message.endpoint,
-        method: message.method,
-        connectionId: message.connectionId,
-        responseType: OpenMethodStreamResponseType.invalidArguments,
-      );
-    }
-
-    List<StreamParameterDescription> requestedInputStreams;
-    try {
-      requestedInputStreams = EndpointDispatch.parseRequestedInputStreams(
-        descriptions: endpointMethodConnector.streamParams,
-        requestedInputStreams: message.inputStreams,
-      );
-    } catch (e) {
+    } on InvalidParametersException catch (e) {
       server.serverpod.logVerbose(
-        'Failed to parse input streams for open stream request: $message',
+        'Failed to parse parameters or input streams for open stream request: $message (${e.message})',
```

Comment:
Question: How is this formatted now? Does the `InvalidParametersException` contain more details around what could not be parsed?

Reply:
Yes, exactly!

Reply:
I assumed we still wanted the `$message` info, so I added the actual exception message inside parentheses.

Reply:
Great!